cobigen-angular-client-generation/index.asciidoc

```diff
@@ -0,0 +1,73 @@
+= Generate Angular Client with devonfw
+== Angular client generation with CobiGen
+====
+With the CobiGen code generator you can easily generate components for your Angular application.
+The following tutorial provides an example for using the CobiGen cli.
+You can find more information on https://github.com/devonfw/cobigen/wiki/howto_angular-client-generation
+## Prerequisites
+* devonfw Ide
+
+## Learning goals.
+Here in this tutorial we will learn the following:
+* Create java project
+* CobiGen to generate the Angular source code out of the java files
+
+====
+
+[step]
+--
+restoreDevonfwIde(["java", "mvn", "vscode"])
+--
+
+For this tutorial an installation of the devonfw IDE is required, which was already set up for you. The next step is to install the CobiGen CLI.
```

Comment:
You should say that this is only required the first time you use it

Reply:
> You should say that this is only required the first time you use it

Added  in the above line..